### ISSUE TITLE
pot-update fails on Debian Unstable "po4a-gettextize ... provide the same amount of master files and localized files"

### ISSUE THREAD

**Issue by stevecotton** [wesnoth/wesnoth#7149]
### Game and System Information

- **Version**: 1.16.6, 1.17.10. The bug is a change in toolchain behavior, so previously-working versions have broken.
- **OS:** Debian Unstable, with po4a version 0.68-1
- **Bug logged with the OS vendor:** https://bugs.debian.org/1020821

### Description of the bug

Trying to build the translations (`scons pot-update`) on an up-to-date Debian Linux install fails with:
```
/usr/bin/po4a-gettextize -f man -m doc/man/wesnoth.6 -m doc/man/wesnothd.6 -p po/wesnoth-manpages/wesnoth-manpages.pot
You must provide the same amount of master files and localized files to synchronize them, as po4a-gettextize is intended to synchronize master files and previously existing translations. If just want to extract POT files of your master files, please use po4a-updatepo. Please note that the most convenient way of using po4a is to write a po4a.conf file and use the integrated po4a(1) program.
```

This is a toolchain change, which has also affected the debian-reference and maint-guide packages. The decision there seems to be that we're using po4a in an unsupported way, but that it's easy to change to using a supported way instead.

**Comment by loonycyborg** [comment]
I'm getting that too now. Will look into fixing this. Hopefully our workflow isn't incompatible with new po4a way.

**Comment by stevecotton** [comment]
Fixed by the combination of #7648 and #7649.
